diff --git a/node_modules/coingecko-api-v3/dist/Interface.d.ts b/node_modules/coingecko-api-v3/dist/Interface.d.ts
index 33f54e2..b9839b0 100644
--- a/node_modules/coingecko-api-v3/dist/Interface.d.ts
+++ b/node_modules/coingecko-api-v3/dist/Interface.d.ts
@@ -54,6 +54,13 @@ export interface TrendingItem {
     market_cap_rank?: number;
     thumb?: string;
     large?: string;
+    data?: {
+      price?: string;
+      price_change_percentage_24h?: {
+        usd: number;
+      },
+      market_cap?: string;
+    };
     score?: number;
 }
 export interface Trending {
